includes/fbproduct.php

```diff
@@ -662,6 +662,31 @@ public function prepare_product( $retailer_id = null, $type_to_prepare_for = sel
 			$brand = wp_strip_all_tags( WC_Facebookcommerce_Utils::get_store_name() );
 		}
 
+		$custom_fields = array(
+			'description' => '',
+			'price' => '',
+			'image_url' => ''
+		);
+
+		// check if fb specific description exists
+		$fb_description = get_post_meta($this->id, self::FB_PRODUCT_DESCRIPTION, true);
+		if (!empty($fb_description)){
+			$custom_fields['description'] = $fb_description;
```

Comment:
Since 'description' already present as the main attribute, let's give more specific names to the data we are sending: 

```suggestion
			$custom_fields['fb_override_description'] = $fb_description;
```

The same for price and image url.